src/core/index.ts

```diff
@@ -1,23 +1,37 @@
 import Store from "./Store";
 import EnvironmentAdapter from "./EnvironmentAdapter";
 import Collection from "./Collection";
+import {
+  OrbitDBOptions,
+  StoreOptions,
+  DatabaseConfigOptions,
+} from "./interfaces";
 const { Key } = require("interface-datastore");
 let datastore = EnvironmentAdapter.datastore(EnvironmentAdapter.repoPath());
 
 class AvionDB extends Store {
   static Collection = Collection;
 
-  static async init(name, ipfs, options: any = {}, orbitDbOptions) {
+  static async init(
+    name: string,
+    ipfs: any,
```

Comment:
Same here - I think you can confidently make `ipfs`  of the IPFS type

Reply:
Same as above. How should I add IPFS type without importing IPFS in the file?